web/js/chatView.js

```diff
@@ -89,8 +91,20 @@
     chatContainer = chatWndElem.querySelector('#chatMsgs');
     chatContent = chatContainer.querySelector('ul');
     chatForm = chatWndElem.querySelector('#chatForm');
+    emojiPicker = document.querySelector('emoji-picker');
+    toggleEmojiBtn = chatWndElem.querySelector('#addEmoji');
   }
 
+  const onEmojiClicked = event => chatMsgInput.value += (" "+(event.detail.unicode)+" ")
+  const toggleEmojiView = evt => {
+      evt.preventDefault();
+    if (emojiPicker.style.display == "none") {
```

Comment:
Please compare using `===`

Reply:
Agreed and implemented changes as per suggestion